cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/ProgramController.java

```diff
@@ -115,7 +121,9 @@ public ProgramRunStatus getRunStatus() {
 
   ListenableFuture<ProgramController> resume();
 
-  ListenableFuture<ProgramController> stop();
+  ListenableFuture<ProgramController> complete();
```

Comment:
We still need `complete` method right? It is called from `ProgramControllerServiceAdapter.terminated` to make the state change.